pkg/config/static_manager_test.go

```diff
@@ -48,7 +48,9 @@ func TestNewStaticProjectConfigManagerFromPayload(t *testing.T) {
 
 func TestStaticGetOptimizelyConfig(t *testing.T) {
 
-	mockDatafile := []byte(`{"accountId":"42","projectId":"123","version":"4"}`)
+	// since marshalling changes the order of keys in jsonString everytime, using json with a single key
```

Comment:
why changed the payload, we are not supposed to change it.

Reply:
reverted this change.